console/billing/how-to/add-billing-contact.mdx

```diff
@@ -0,0 +1,38 @@
+---
+meta:
+  title: How to add a billing contact
+  description: This page explains how to add a billing contact
+content:
+  h1: How to add a billing contact
+  paragraph: This page explains how to add a billing contact
+tags: billing payment contact
+dates:
+  validation: 2024-02-16
+  posted: 2024-02-16
+categories:
+  - billing
+---
+
+The **billing contact** is an additional contact linked to your account who will receive your monthly invoices. This can be useful if invoices in your Organization are handled by another department. 
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```

Comment:
Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.